packages/lab/package.json

```diff
@@ -62,6 +62,7 @@
     "@types/react-dom": "^18.2.5",
     "@types/react-grid-layout": "^1.3.5",
     "npm-run-all": "^4.1.5",
+    "react-error-boundary": "^4.0.11",
```

Comment:
This should be in root's `devDependencies`, as it's used for docs/stories and not the "package" itself. It's a bit strange, but it's what we're doing elsewhere

Reply:
Thanks, updated the PR